rcl/src/rcl/graph.c

```diff
@@ -478,12 +478,27 @@ _rcl_wait_for_entities(
   RCL_CHECK_ARGUMENT_FOR_NULL(topic_name, RCL_RET_INVALID_ARGUMENT);
   RCL_CHECK_ARGUMENT_FOR_NULL(success, RCL_RET_INVALID_ARGUMENT);
 
+  char expanded_topic[256];  // topics can't be longer than 256 characters
```

Comment:
Is this declared as a constant (or `#define`) somewhere? `RCL_MAX_TOPIC_LENGTH` could be good to have